feat: clang-tidy check google-runtime-int

yaoyinnan opened the pull request:
Replace all integer types (short int long etc) with (int16_t int32_t int64_t etc).

longfar-ncy commented:
考虑把 .clang-tidy 文件第28行对该检查项的屏蔽删掉？

AlexStocks commented on src/net/src/net_util.cc:
```diff
@@ -12,8 +12,8 @@
 
-int Setnonblocking(int sockfd) {
-  int flags;
+int32_t Setnonblocking(int32_t sockfd) {
```
sockfd 这种 fd，一般还是保持 int 比较合适吧？

longfar-ncy reviewed (commented):
没问题后考虑把 .clang-tidy 文件中第28行内容删掉再检测一下试试

yaoyinnan commented:
> 没问题后考虑把 .clang-tidy 文件中第28行内容删掉再检测一下试试

打开 google-runtime-int 检查并且都通过了

yaoyinnan committed:
fix: clang-tidy check google-runtime-int

Replace all integer types (short int long etc) with (int16_t int32_t int64_t etc).

Signed-off-by: yaoyinnan <yaoyinnan@foxmail.com>
Signed-off-by: yaoyinnan <35447132+yaoyinnan@users.noreply.github.com>

AlexStocks commented:
3.5 和 unstable 不再做大改，所以这个 PR 不再需要。
